app/models/payment.rb

```diff
@@ -8,7 +8,23 @@ class Payment < ApplicationRecord
   validates :secure_number, length: { is: 3, allow_blank: true }
   validates :payment_type, presence: true, inclusion: { in: payment_types.keys }
 
-  def full_name
-    "#{first_name.capitalize} #{last_name.capitalize}".strip
+  def payment_array
+    [payment_type, id]
+  end
+
+  def self.uncheck_active_method
+    active.first.current_method = false
+  end
+
+  def current_method?
+    current_method
+  end
+
+  def type_info
+    if %w[credit_card debit_card].include? payment_type
+      card_number.to_s.last(4).to_i
```

Comment:
what happens if my CC ends in 0021? to_i is going to display 21 right? Maybe the to_i is not necessary

Reply:
Commit: b5b13dd